wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrationVersion.swift

```diff
@@ -130,8 +131,10 @@ enum CoreDataMessagingMigrationVersion: String, CaseIterable {
 
     var nextVersion: Self? {
         switch self {
-        case .version2_109:
+        case .version2_110:
             return nil
+        case .version2_109:
+            return .version2_110
         case .version2_108:
             return .version2_109
```

Comment:
Theoretically we could use the behaviour core data had before to go directly from `108` to `110` without breaking anything. But to be honest I would rather keep this step by step migration and it was also merged this way to the `release` branch.